ext/iperf3/iperf_udp.c

```diff
@@ -593,14 +727,29 @@ iperf_udp_connect(struct iperf_test *test)
     }
 
 #if defined(CONFIG_NRF_IPERF3_INTEGRATION)
+    struct timeval tv;
+
 #ifdef SO_RCVTIMEO
     /* 30 sec timeout for a case when there is a network problem. */
```

Comment:
It would be better to set the timeout value before the `SO_RCVTIMEO` flagging. If `SO_SNDTIMEO` would be defined and `SO_RCVTIMEO` not, the timeout value would be uninitialized when setting the `SO_SNDTIMEO`. Although probably that will never be the case. 😃

Reply:
Changed.